plugins/woocommerce/includes/class-wc-ajax.php

```diff
@@ -3636,6 +3637,20 @@ public static function toggle_gateway_enabled() {
 		wp_die();
 	}
 
+	/**
+	 * AJAX handler for asynchronously loading the status widget content.
+	 */
+	public static function load_status_widget() {
+		check_ajax_referer( 'wc-status-widget', 'security' );
```

Comment:
We should add the capability checks from `WC_Admin_Dashboard::should_display_widget()` here as well.

Reply:
💯 Added as part of https://github.com/woocommerce/woocommerce/pull/56039/commits/c9e0ba2a9ac482de6be0376ca0e51c3fd2e53fa7